Update activation instructions in activation.md (#71)
AdguardTeam/KnowledgeBase: docs/general/license/activation.md

## How to activate a license

> To activate your license, go to the *License* section of your AdGuard product and find the *Log in* field or *Activate* button. Then use the email and password of your AdGuard account to log in. If you don't have an account yet, you will be offered to create one.

Reviewer: я бы написала просто Use the email and password of your AdGuard account to log in

> > Note: For successful activation using AdGuard account, you need to have a valid license with at least one open slot there. (If your license is already used on a maximum number of devices, you can [upgrade it](#how-to-upgrade-a-license) or [transfer key from one device to another](#how-to-transfer-a-license-to-another-device).)

Reviewer: ```suggestion
> Note: To activate a license via AdGuard account, you need to have a valid license with at least one open slot there. If your license is already used on a maximum number of devices, you can [upgrade it](#how-to-upgrade-a-license) or [transfer it from one device to another](#how-to-transfer-a-license-to-another-device).
```

Alternatively, you can use a [license key](../what-is#license-key) to activate the license. 

**Below you can check step-by-step activation instructions for each platform.**

### Activating AdGuard for Windows

1. Open the AdGuard app
   

> To do it, doubleclick on the AdGuard icon on the desktop:   

Reviewer: double-click the AdGuard icon

![AdGuard icon on the desktop](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowsEn.png)

> or double click on the AdGuard icon in the system tray:   

Reviewer: double-click the AdGuard icon in the system tray menu

![AdGuard icon in the system tray](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windows2En.png)

> 2. Go to **License** in the main menu, then click **Activate license** 

Reviewer: ```suggestion
2. Go to **License** tab in the app menu and click **Activate license**
```

![Click Activate license](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense1en.png)

3. You will be presented with two options for activation:

> - Via entering the **email** and **password** from your **AdGuard  account** (or through a FB/Google/Apple account)

Reviewer: AdGuard account (лишний пробел) + осталось `a FB/Google/Apple account`, так не очень

> - Via entering **Email** and **Password** from your **AdGuard  account** (or through a FB/Google/Apple account)

Reviewer: ```suggestion
 Via entering the email and password from your **AdGuard  account** (or from your Facebook, Google, or Apple account)
```

![Log in to your AdGuard account](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense2en.png)

- Via entering the **license key**

![Enter license key](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/windowslicense3en.png)

### Activating AdGuard for Mac

> 1. Open AdGuard app (lick on the AdGuard icon in the menu bar) 

Reviewer: ```suggestion
1. Open the AdGuard app (click the AdGuard icon in the menu bar)
```

![AdGuard icon in the menu bar](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/Ja/General/mac1.png)

> 2. Click on **gear icon** in the top right corner

Reviewer: ```suggestion
2. Click the **gear icon** in the top right corner
```

3. Click **License...** 

![Go to Settings and License](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/macEn.png)

> 4. In the opened window press **Activate** button   

Reviewer: press **Activate** / press the **Activate** button

![Press Activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen1.png)

5. You will be presented with two options for activation:

> - Activate by entering **Email** and **Password** from your **AdGuard  account** (or through a FB/Google/Apple account)

Reviewer: Via entering the email and password from your **AdGuard  account** (or from your Facebook, Google, or Apple account)

Reviewer: и тут

![Log in to your AdGuard account to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen2.png)

> - Activate with a **license key**

Reviewer: Via entering the **license key**

Author: *не уверен что the, мы же не про какой-то конкретный ключ говорим

Reviewer: ну значит а, но давай везде единообразно. выше было Via entering

![Enter your license key to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen3.png)

### Activating AdGuard for Android

1. Open the AdGuard app
2. Tap **menu icon (☰)** in the top left corner of the screen
3. Tap **Get Premium** bar (or tap **License Status**)

![Tap Get Premium or License Status](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/androidlicense1en.png)

> 4. Tap **three dots menu (⋮)** in the top right corner of the screen

Reviewer: three-dot menu

> ![Tap three dot menu](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/android2En.png)

Reviewer: three-dot

Author: в гугле и так и так пишут, я оттуда взял

5. Press **Activate Premium**, and you will be presented with two activation options:

> - Enter **Email** and **Password** from your **AdGuard account** (or sign in with Facebook/Google account)

Reviewer: Via entering the email and password from your AdGuard account (or from your Facebook, Google, or Apple account)

![Log in to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/androidlicense2en.png)

> - Or enter a **license key**

Reviewer: Via entering the **license key**

Author: не уверен что the, мы же не про какой-то конкретный ключ говорим

Reviewer: тогда здесь тоже поправь на a: https://uploads.adguard.org/el.termikael9tx32.png

![Enter your license key to activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/androidlicense3en.png)

### Activating AdGuard for iOS

> 1. Open the application and tap on the **gear icon** in the bottom right corner of the screen

Reviewer: ```suggestion
1. Open the AdGuard app and tap the **gear icon** in the bottom right corner of the screen
```

2. Tap **License**

3. Tap **Log in** in the top right corner of the screen

![Tap Log in in the top right corner of Premium screen](https://cdn.adguard.com/content/kb/ad_blocker/iOS/ioslicense1en.png)

> 4. Enter **Email** and **Password** from your **AdGuard account**

Reviewer: зачем с больших?

Author: не знаю, так в старой инструкции было. ну вообще названия полей так выглядят (с большой), поэтому так легче мб просто визуально увидеть что надо заполнять. это как названия полей с которыми юзер должен взаимодействовать

Reviewer: ну хз насколько легче...

![Press Activate](https://cdn.adguard.com/public/Adguard/kb/newscreenshots/En/General/maclicenseen1.png)

5. You will be presented with two options for activation:

> - Via entering **email** and **password** from your **AdGuard  account** (or from your Facebook/Google/Apple account)

Reviewer: the **email** and **password**
